src2/Components/Header/HeaderLower/HeaderLower.tsx

```diff
@@ -0,0 +1,80 @@
+import { Grid, Box, Button, Backdrop } from "@mui/material";
+import CssBaseline from "@mui/material/CssBaseline";
+import React, { useState } from "react";
+import HomeIcon from "@mui/icons-material/Home";
+import AddIcon from "@mui/icons-material/Add";
+import { CustomText } from "./style";
+import ModalAppProjectUpper from "../ModalAddProject/ModalAddProject";
+export interface HeaderLowerProps {}
+const HeaderLower: React.VFC<HeaderLowerProps> = ({}) => {
+  const [openModalAddProject, setOpenModalAddProject] = useState(false);
+
+  const handleToggleModalAddProject = () => {
+    setOpenModalAddProject(!openModalAddProject);
+  };
```

Comment:
```suggestion
 const useToggleModalAddProject = () => {
    setOpenModalAddProject(!openModal);
  };
```
can you fix it like this?